.circleci/config.yml

```diff
@@ -303,7 +303,7 @@ workflows:
       - linux_integration_tests:
           matrix:
             parameters:
-              kitchen_target: ["bitbar-ubuntu-1804", "bitbar-ubuntu-2204", "linux-ubuntu-1804"]
+              kitchen_target: ["bitbar-ubuntu-2204", "linux-ubuntu-1804"]
```

Comment:
we should have linux-ubuntu-2204 in here

Reply:
Good catch!